parlai/crowdsourcing/tasks/turn_annotations/README.md

```diff
@@ -0,0 +1,26 @@
+# THIS TASK TRANSITION IS STILL A WIP
+
+# Turn Annotations MTurk Task
+
+This task will collect conversations between a human and a model. After each response by the model, the human will be prompted to annotate the model's response by selecting  checkboxes that represent customizable attributes of that response.
+
+## Launching
+
+**Python launch script**: see `launch.py` for an example script.
```

Comment:
Nit: this is `run.py`, not `launch.py`, right?

Reply:
Indeed I haven't updated the README at all.